components/Collections/src/MDCCollectionViewController.m

```diff
@@ -345,12 +338,19 @@ - (BOOL)collectionView:(UICollectionView *)collectionView
 
 - (void)collectionView:(UICollectionView *)collectionView
     didHighlightItemAtIndexPath:(NSIndexPath *)indexPath {
-  // Start cell ink show animation.
-  MDCInkView *inkView =
-      [self inkTouchController:_inkTouchController inkViewAtTouchLocation:_inkTouchLocation];
+
   UICollectionViewCell *cell = [collectionView cellForItemAtIndexPath:indexPath];
   CGPoint location = [collectionView convertPoint:_inkTouchLocation toView:cell];
 
+  // Start cell ink show animation.
```

Comment:
Usually after calling [object respondsToSelector:(methodX)], I would expect us to invoke [object methodX].
Does the fact that cell responds to inkView indicate something about self that lets us invoke inkTouchController:inkViewAtTouchLocation?
Should this be inkView = cell.inkView?